### ISSUE TITLE
Very small cursor (on Linux)

### ISSUE THREAD

**Issue by Artim96** [upscayl/upscayl#965]
### Checklist

- [X] I have **tried ALL** of the steps mentioned in the [troubleshooting guide](https://github.com/upscayl/upscayl/wiki/Troubleshooting).
- [X] I have **searched** for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Describe the Bug

While it's only a cosmetic issue, but on a high resolution display, such a small cursor is very difficult to see. Also I have no idea what UI toolkit is used and setting the XCURSOR_SIZE environment variable via Flatseal didn't seam to help.

### To Reproduce

Use Upscayl on Linux (Gnome 46, Wayland, fractional scale set to 125 %) installed from Flathub

### Upscayl Version (or commit hash)

2.11.5

### Platform

Linux

### OS Version

Debian GNU/Linux trixie/sid

### GPU Name

Radeon 780M

### Expected Behavior

Show normal sized cursor

### Screenshots

![screenshot-upscayl](https://github.com/user-attachments/assets/58252761-b843-4e2c-bed6-0282d9091073)

### Logs

```sh
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"Digital Art","value":"realesrgan-x4plus-anime"}
⚙️ Getting gpuId from localStorage:  0
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus-anime
⚙️ Getting saveImageAs from localStorage:  png
⚙️ Getting model from localStorage:  {"label":"Digital Art","value":"realesrgan-x4plus-anime"}
⚙️ Getting gpuId from localStorage:  0
```

**Comment by aaronliu0130** [comment]
Do other Flatpak apps not have this issue?

**Comment by Artim96** [comment]
None that I've ever used. Especially no other Electron apps, though I don't know of any other app that uses React.

**Comment by NayamAmarshe** [comment]
Very strange. I suspect a fractional scaling issue.

**Comment by Artim96** [comment]
Would be my guess too. The bot already gave some helpful hints. The cursor icon does look a lot like it's being cropped. Is it possible to edit the variables in the installed Flatpak and if so how? Or would they have to be changed in source?

I won't be able to do any testing for the next week, but I already did some testing with chrome flags, as they sometimes help with issues with Electron apps. Sadly to no avail.

**Comment by NiffirgkcaJ** [comment]
I have this issue too! I'm currently running GNOME 47 and have this installed as a Flatpak; the cursor smaller than normal in 100% and becomes tiny in 125%, etc.

**Comment by Artim96** [comment]
As there hasn't been an update since, it's still happening.

**Comment by Artim96** [comment]
So, I've got good news and bad news with yesterdays 2.15 release. The bad news: the issue is still present. The good news, I've also tested the AppImage version this time. It does default to XWayland, but with `--ozone-platform=wayland` it will use Wayland. And scaling does work with it properly. So whatever the Flatpak version does differently, it seems to only affect it.

Also, if that helps, I did also copy the logs from the flatpak version:
`💻 System Info: {"platform":"linux","release":"6.12.6-amd64","arch":"x64","model":"AMD Ryzen 7 7840HS w/ Radeon 780M Graphics","cpuCount":16,"gpu":{"active":true,"cudaComputeCapabilityMajor":0,"deviceId":5567,"gpuPreference":0,"vendorId":4098}}`
